guides/common/modules/proc_importing-syncable-exports.adoc

```diff
@@ -0,0 +1,12 @@
+[id="Importing_Syncable_Exports_{context}"]
+= Importing Syncable Exports
```

Comment:
Should we specify what are the syncable exports in this context?

Reply:
Maybe. Before we get this merged, let me get a QE check on this PR.

Reply:
Syncable exports are obtained by running the ```hammer content-export --format=syncable' command```.

However, these get added into the /var/lib/pulp/exports directory.

In order to use the ```hammer content-import``` command, that syncable export needs to be moved or copied into /var/lib/pulp/imports

Once that exported repository/contentview is in /var/lib/pulp/imports, you can then run your ```hammer content-import``` command

EX:

```hammer content-export complete repository --id='' --format=syncable```
```mv /var/lib/pulp/exports/<continued path to exported repo> var/lib/pulp/imports```
```hammer content-import repository --org='' --path=/var/lib/pulp/imports/<exportedrepo>```

I think we either need to reference a chapter about sycnable exports, or add that any export needs to be added into the /var/lib/pulp/imports directory before the ```hammer content-import command``` is run. 

If we do not have detailed steps in any chapters we can link. Feel free to let me know and I can type something up.

Reply:
"In order to use the hammer content-import command, that syncable export needs to be moved or copied into /var/lib/pulp/imports" 
This is mostly correct.  You can copy it in any of the sub-directories marked as `ALLOWED_IMPORT_PATHS` in `/etc/pulp/settings.py`.  Its just that `/var/lib/pulp/imports` is one of the sub-directories in the `ALLOWED_IMPORT_PATHS` which is set by installer on default.

Reply:
@ColeHiggins2 We are still working on chapters covering syncable exports and imports for 6.13. Is there something you want to write up for me to put in this PR? Or do you think what you have mentioned in your comment here will suffice?